src/main/java/uk/gov/cabinetoffice/csl/domain/User.java

```diff
@@ -37,16 +37,25 @@ public static User fromUserDetails(String uid, UserDetailsDto userDetailsDto) {
     }
 
     public String getFormattedOrganisationName() {
+        if(departmentHierarchy == null) {
```

Comment:
Not required with null-safe empty arraylist initialization, same for the other null checks

Reply:
Done